packages/data-service/index.js

```diff
@@ -0,0 +1,2 @@
+export { graphqlObservable } from "./src/graphqlObservable";
```

Comment:
Can you pull this move of data-service into a seperate commit and explain why you need to do this?

Reply:
I think that fixes an importing problem bill was having. I will try to figure out, but I don`t really know.